packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -253,8 +248,12 @@ export class VerticalStackedBarChartBase extends React.Component<
     return lineLegends;
   };
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  private _createLines = (xScale: any, yScale: any, containerHeight: number, containerWidth: number): JSX.Element => {
+  private _createLines = (
+    xScale: NumericScale,
```

Comment:
it can also be dateScale

Reply:
For strings, it still uses a numeric scale using the indexes. It cannot be dateScale